src/configurations/crc-researcher/handleParticipantWorkflowChange.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react'
```

Comment:
can remove this import and make this file .ts instead, sometimes typescript decides unused imports should break the build (and other times it doesn't :) )